File: CursoPOOUber/Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {

    Map<String, ArrayList<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, String driver, String brand, String model,
            Map<String, ArrayList<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }

}

File: CursoPOOUber/PHP/uberBlack.php
<?php
require_once('Cars.php');

class uberBlack extends Car
{
    public $tipeCarAccepted;
    public $seatMaterial;

    public function __construct($license, $driver, $tipeCarAccepted, $seatMaterial)
    {
        parent::__construct($license, $driver);
        $this->tipeCarAccepted = $tipeCarAccepted;
        $this->seatMaterial = $seatMaterial;
    }
}
